### DISCUSSION TITLE
WalletConnect v1.0 will be officially deprecated on March 1, 2023

### DISCUSSION THREAD

**Discussion by Dmytro-Kyparenko** [NoahZinsmeister/web3-react#707] [General]
WalletConnect has [announced](https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e) that it is moving to version 2.0 and WalletConnect v1.0 will be officially deprecated on March 1, 2023.
Do you plan to upgrade to the new version?

**Comment by alysiawu** [comment]
asking the same question here as well
